lua/nvim-treesitter/install.lua

```diff
@@ -243,6 +243,7 @@ local function do_download_tar(logger, repo, project_name, cache_dir, revision,
 
   logger:debug('Creating temporary directory: ' .. temp_dir)
   --TODO(clason): use vim.fn.mkdir(temp_dir, 'p') in case stdpath('cache') is not created
+  util.delete(temp_dir)
```

Comment:
Aren't you already deleting that dir in line 221?

Reply:
Yeah, I didn't understand the issue when I added this.